### PR TITLE
Pre-launch design changes for connected accounts.

### PR DESCRIPTION
**Pull request by dvogel**
## Description

This cleans up some text wording and formatting in accordance with the pre-launch checklist review. See [vets-contrib ticket #1121](https://github.com/department-of-veterans-affairs/vets-contrib/issues/1121) and [the design mockups](https://www.figma.com/proto/QHtcfdUtB4N4lZZDXQWRNR/Auth-Management-Connected-Acct?node-id=245%3A2&viewport=-2303%2C-2468%2C1.16021&scaling=min-zoom) for more details.

One aspect of the design mockups I didn't make is the list bullet point changes. The site seems to lack any disc bullet lists. The list bullet default in the brand consolidation CSS is explicitly square, so that seemed like an oversight in the mockups. @juliaelman please let me know if whether this is right or wrong.

## Testing done

Only manual testing in chromium so far. I'll be following up [with manual testing](https://github.com/department-of-veterans-affairs/vets-external-teams/blob/master/DeveloperDocs/testing/cross-browser-manual-testing.md) once I figure out how to gain access to the staging environment.

## Acceptance criteria
- [x] Incorporate all changes from the design mockups.

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: src/applications/personalization/account/components/ConnectedAccountsSection.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import isBrandConsolidationEnabled from '../../../../platform/brand-consolidation/feature-flag';
```
Comment (erikphansen) [inline src/applications/personalization/account/components/ConnectedAccountsSection.jsx]: We no longer need to use this utility. It’s VA.gov all the time now. Moving forward if you come across usage of this utility in files you are changing feel free to pull its usage out.

File: src/applications/personalization/account/components/ConnectedAccountsSection.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+const propertyName = isBrandConsolidationEnabled() ? 'VA.gov' : 'Vets.gov';
+
+export default function ConnectedAccountsSection() {
+  return (
+    <div>
+      <h3>Connected accounts</h3>
+      <div>
+        <p>
+          Manage the settings for the sites and applications you've given
```
Comment (erikphansen) [inline src/applications/personalization/account/components/ConnectedAccountsSection.jsx]: Could you make this a curly single quote?

File: src/applications/personalization/connected-accounts/components/AccountModal.jsx [hunk 3]
```diff
@@ -7,20 +7,19 @@ export function AccountModal({
...
       onClose={onCloseModal}
-      title={`Are you sure you want to disconnect from ${appName}?`}
+      title={`Please confirm you want to disconnect from ${appName}`}
       visible={modalOpen}
     >
       <p>
-        {appName} will no longer be able to access your {propertyName} account
-        informaton. You cannot undo this action.
+        {appName} won't be able to access your profile data once you disconnect.
+        You can't undo this action.
```
Comment (erikphansen) [inline src/applications/personalization/connected-accounts/components/AccountModal.jsx]: Plus update these to be curly single quotes as well

File: src/applications/personalization/connected-accounts/components/ConnectedApps.jsx [hunk 4]
```diff
@@ -7,11 +7,13 @@ export function ConnectedApps({ confirmDelete, propertyName, accounts }) {
...
       <div className="usa-width-two-thirds medium-8 small-12 columns">
-        <h1>Your Connected Accounts</h1>
+        <h1>Connected Accounts</h1>
         <p className="va-introtext">
           {/* eslint-disable prettier/prettier */}
-          You gave these sites and applications read only access to some of
-          your {propertyName} account data.
+          You gave the sites and applications listed below permissions to
+          access some of your {propertyName} profile data. These sites and
+          applications can only view your information. They can't change
```
Comment (erikphansen) [inline src/applications/personalization/connected-accounts/components/ConnectedApps.jsx]: Another curly single please

File: src/applications/personalization/connected-accounts/components/ConnectedApps.jsx [hunk 5]
```diff
@@ -47,8 +47,9 @@ export function ConnectedApps({ confirmDelete, propertyName, accounts }) {
               })
             }
           >
-            Go to {propertyName} FAQs
+            Go to connect account FAQs
```
Comment (erikphansen) [inline src/applications/personalization/connected-accounts/components/ConnectedApps.jsx]: Is this right?

File: src/applications/personalization/connected-accounts/components/ConnectedApps.jsx [hunk 6]
```diff
@@ -30,13 +32,11 @@ export function ConnectedApps({ confirmDelete, propertyName, accounts }) {
...
 
         <div className="feature">
-          <h3>Have questions about signing in to {propertyName}?</h3>
+          <h3>Have questions about connected accounts?</h3>
           <p>
-            Get answers to frequently asked questions about how to sign in,
-            common issues with verifying your identity, and your privacy and
-            security on {propertyName}.
+            Get answers to frequently asked questions about connected accounts
+            and how they work with your {propertyName} account.
```
Comment (erikphansen) [inline src/applications/personalization/connected-accounts/components/ConnectedApps.jsx]: Looks like there’s an opportunity to hardcore everything to VA.gov in this file, too

File: src/applications/personalization/connected-accounts/components/ConnectedApps.jsx (hunk 5 above)
Comment (dvogel) [reply src/applications/personalization/connected-accounts/components/ConnectedApps.jsx]: It was supposed to be "connected account". I just pushed a fix. Thanks for catching that.

**Comment by jaredcunha-usds** [comment]
For the bullets, I'm fine with using the square bullets instead of of the discs since that is what the default CSS is serving. I don't think it would be worth creating a separate class for the round bullets… I think it's just an issue of Sketch/Figma typically using discs by default. @juliaelman, let me know if I'm incorrect here.

**Comment by juliaelman** [comment]
@jaredcunha-usds correct. Are there plans to switch the default to discs?

**Comment by jaredcunha-usds** [comment]
@juliaelman Not that I know of. Seems that the discs are provided by USWDS by default, and at some point, we've made a conscious decision to use the squares.

**Comment by dvogel** [comment]
@juliaelman Ready for you review again.

**Comment by dvogel** [comment]
@juliaelman Yeah, I forgot login doesn't work on the preview sites. Here's some screenshots:

* [account page](https://user-images.githubusercontent.com/45699/52351683-de4d2b80-29f0-11e9-8b86-b1d71c8212d3.png)
* [connect accounts initial](https://user-images.githubusercontent.com/45699/52351690-e016ef00-29f0-11e9-93b4-fe8b659b7233.png)
* [connected accounts expanded](https://user-images.githubusercontent.com/45699/52351697-e3aa7600-29f0-11e9-8de7-65172a7cd49d.png)
* [connected accounts dialog](https://user-images.githubusercontent.com/45699/52351701-e5743980-29f0-11e9-8571-9791a1fac0cd.png)

**Comment by juliaelman** [comment]
@dvogel looks GTM, but it looks like there are some inconsistencies in the banner for the personalized homepage :\

@Samara-Strauss when you have a moment, would you mind taking a look at the this and letting us know where the `NEW: check out your new personalize homepage` banner should go? Want to make sure it is appearing correctly.

**Comment by Samara-Strauss** [comment]
> @Samara-Strauss when you have a moment, would you mind taking a look at the this and letting us know where the `NEW: check out your new personalize homepage` banner should go? Want to make sure it is appearing correctly.

So this isn't specifically an announcement for the personalized homepage (My VA) — it's the state we use when we have feature or beta announcements we want to communicate about the site. If I recall correctly, that banner should be anchored to the bottom of someone's window until the X out of it, but it might be best to confirm that by looking in the code.

**Comment by juliaelman** [comment]
Thanks Samara!

@dvogel is that context helpful?

**Comment by dvogel** [comment]
@juliaelman That banner is fixed to the bottom of the window in the browser. When my screenshot-entire-page extension scrolled down to the bottom the banner didn't move. When scrolled in a real user session it does though.

**Comment by juliaelman** [comment]
@dvogel gotcha. :shipit:

**Review by juliasolorzano — APPROVED** [review]

**Comment by dvogel** [comment]
Manual cross-browser testing complete. I tracked my work using the template: [dv-1121-template-cross-browser-manual-testing.xlsx](https://github.com/department-of-veterans-affairs/vets-website/files/2847050/dv-1121-template-cross-browser-manual-testing.xlsx). Only one small issue was found, unrelated to the changes in the mockups (verified by going back to a previous commit). On mobile chrome and mobile safari, when the connected app scopes are expanded or collapsed, the viewport snaps to the top of the view, causing the user to scroll down again. @juliaelman any idea whether this is a known issue with va.gov? Either way it seems unrelated to this ticket to me so I think it's ready for your sign-off.

**Comment by juliaelman** [comment]
@dvogel I am not sure if this is a known issue or not. When you have a moment, would you mind providing a screencast of the issue you are seeing? I'd like to be able to show the VA.gov team so that they are aware of the issue and see if it is site wide.

**Pull request merged**